R/style_roc.R

```diff
@@ -30,9 +31,10 @@ style_roc <- function(major.breaks = c(0, .1, .25, .5, .75, .9, 1),
     
     if(guide){
       
-      pcol <- theme()$panel.grid.major$colour
-      if(is.null(pcol)) pcol <- "white"
-      res <- append(res, geom_abline(slope = 1, intercept = 0, color = pcol))
+      if(is.null(guide_col)) guide_col <- theme()$panel.grid.major$colour
```

Comment:
Sorry for the enter into the conversation, but these `if` conditions are wrong.

Reply:
what do you think is wrong about them?

Reply:
Sorry, it was my bad.